samples/language/Unit.qs

```diff
@@ -1,17 +1,20 @@
-/// # Sample
-/// Unit
-///
-/// # Description
-/// The `Unit` type is the singleton type whose only value is ().
-/// Functions implicitly return `Unit` if no explicit or implicit
-/// return is specified.
-namespace MyQuantumApp {
+// # Sample
+// Unit
+//
+// # Description
+// The `Unit` type is the singleton type whose only value is ().
+// Functions implicitly return `Unit` if no explicit or implicit
```

Comment:
Functions won't return Unit except for in these samples, since a function returning Unit would not be useful (no side effects like operation can have). I'd generalize
```suggestion
// Callables implicitly return `Unit` if no explicit or implicit
```